src/site/content/en/installable/discover-installable/index.md

```diff
@@ -1,102 +0,0 @@
----
```

Comment:
This content is all covered in the general PWA content and is no longer necessary.